docs/release/sending-announcements.md

```diff
@@ -0,0 +1,99 @@
+            yum install git python-requests python-gnupg
+
+    -   Ubuntu
+
+            :::console
+            apt install git python-requests python-gnupg
+
+1.  Install the OSG tools:
+
+        :::console
+        git clone https://github.com/opensciencegrid/topology.git
+
+1.  Disable any HTTP proxies by looking in the environment for the following variables and unset them if present:
+
+        :::console
+        env | grep -i 'https*_proxy'
+        unset HTTPS_PROXY HTTP_PROXY https_proxy http_proxy
+
+1. If you are on a FermiCloud VM, update `postfix` to relay through FermiLab's official mail server:
+
+        :::console
+        echo "transport_maps = hash:/etc/postfix/transport" >> /etc/postfix/main.cf
+        echo "*   smtp:smtp.fnal.gov" >> /etc/postfix/transport
+        postmap hash:/etc/postfix/transport
+        postfix reload
+
+5.  Ensure that you can lookup contacts in the topology database by using the `osg-topology` tool to list the contacts:
+
+        :::console
+        cd topology
+        PYTHONPATH=src python bin/osg-topology --cert publicCert.pem \
+            --key privateKey.pem list-resource-contacts
+
+    If the contacts include email addresses, this is working properly.
+    If you type your password incorrectly, the authentication will silently fail and you won't get email addresses
+
+Sending the announcement
+------------------------
+
+Use the `osg-notify` tool to send the announcement using the relevant options from the following table:
+
+| Option                          | Description                                                                                                  |
+|---------------------------------|--------------------------------------------------------------------------------------------------------------|
+| `--dry-run`                     | Use this option until you are ready to actually send the message                                             |
+| `--cert <FILE>`                 | File that contains your OSG User Certificate                                                                 |
+| `--key <FILE>`                  | File that contains your Private Key for your OSG User Certificate                                            |
+| `--no-sign`                     | Don't GPG sign the message (release only)                                                                    |
+| `--type production`             | Not a test message                                                                                           |
+| `--message <FILE>`              | File containing your message                                                                                 |
+| `--subject <EMAIL SUBJECT>`     | The subject of your message                                                                                  |
+| `--recipients <LIST OF EMAILS>` | (optional) List of recipient email addresses, must have at least one                                         |
```

Comment:
I'm pretty sure this is optional, right?

Reply:
If it's moved there, a link would be nice.

Reply:
You need at least one recipient. It is not optional.

Reply:
Fixed the doc but this seems like strange behavior. I would think one of `--oim-recipients` and `--recipients` is required.